deploy/updater/updater.py

```
@@ -4,329 +4,398 @@
 import logging
 import datetime
 from aiomcache import Client
+from deepdiff import DeepDiff
```

Ruff (0.8.2)

`deepdiff.DeepDiff` imported but unused

Remove unused import: `deepdiff.DeepDiff`

(F401)